File: Programas em Phyton/Ex34.py
numeros = [10]
comparando = 0
for i in range (0, 10, 1):
    x = int(input('Digite os 10 números: '))
    numeros.append(x)
for i in range (0, 10, 1):
    if(numeros[i]>=comparando):
        comparando = numeros[i]
print(f'O maior número é: {comparando}')

File: Programas em C/Ex34.c
#include<stdio.h>
#include<stdlib.h>
#include<ctype.h>
int main(int argc, char const *argv[])
{
    int numeros[10];
    int comparando;
    comparando=0;

    printf("Digite os 10 numeros: ");
    for(int i=0; i<10; i++)
    {
        scanf("%i", &numeros[i]);
    }
    for(int i=0; i<10; i++)
    {
        if(numeros[i] >= comparando)
        {
            comparando = numeros[i];
        }
    }
    printf("O maior numero e: %i", comparando);
    return 0;
}
